src/main/java/org/folio/edge/orders/OrdersHandler.java

```diff
@@ -174,6 +179,7 @@ private void processErrorResponse(RoutingContext ctx, String respBody, String co
   }
 
   private void handleErrorResponse(RoutingContext ctx, int status, ResponseWrapper responseWrapper) {
```

Comment:
I don't see that we log something before invoking handleErrorResponse and we do not logging anything in this method.
Lets add log here with warn level and log response wrapper into it(probably need to override toString for it)

Reply:
logger level will change and toString will be implemented for both ResponseWrapper and ErrorWrapper